.github/workflows/required_approvals.yml

```diff
@@ -0,0 +1,26 @@
+name: PR Approval Workflow
+on:
+  pull_request:
+    branches:
+    - main
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  check-approvals:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      pull-requests: read
+    steps:
+    - name: Check for required approvals
+      id: check-approvals
+      uses: skymoore/required-approvals@main
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        read_org_scoped_token: ${{ secrets.READ_ORG_SCOPED_TOKEN }}
+        org_name: openconfig
+        min_approvals: 2
```

Comment:
say I have 2 user-groups for an entry in the OWNER file. Does it mean 1 LGTM from each group, or 2 LGTM from the same group is ok too? Can you help add some comment here for future readers? Thanks.